nomad/vault.go

```diff
@@ -1391,15 +1408,27 @@ func (v *vaultClient) Stats() map[string]string {
 	stat := v.stats()
 
 	expireTimeStr := ""
-
 	if !stat.TokenExpiry.IsZero() {
 		expireTimeStr = stat.TokenExpiry.Format(time.RFC3339)
 	}
 
+	lastRenewTimeStr := ""
+	if !stat.LastRenewalTime.IsZero() {
+		lastRenewTimeStr = stat.LastRenewalTime.Format(time.RFC3339)
+	}
+
+	nextRenewTimeStr := ""
+	if !stat.NextRenewalTime.IsZero() {
+		nextRenewTimeStr = stat.NextRenewalTime.Format(time.RFC3339)
+	}
+
+
 	return map[string]string{
 		"tracked_for_revoked": strconv.Itoa(stat.TrackedForRevoke),
 		"token_ttl":           stat.TokenTTL.Round(time.Second).String(),
 		"token_expire_time":   expireTimeStr,
+		"last_renewal_time":   lastRenewTimeStr,
+		"next_renewal_time":   nextRenewTimeStr,
```

Comment:
```suggestion
		"token_last_renewal_time":   lastRenewTimeStr,
		"token_next_renewal_time":   nextRenewTimeStr,
```

Some more consistency changes.

Reply:
updated metric names in a separate commit